src/main/java/core/mate/academy/producer/TrackProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.producer;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrackProducer implements MachineProducer {
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
Do not use wildcards when you return something. Wildcards are for input validation.
```suggestion
    public List<T> get() {
```